packages/react/src/components/Authenticator/SignUp/FormFields.tsx

```diff
@@ -70,15 +76,29 @@ export function FormFields() {
             );
           case 'password':
             return (
-              <PasswordField
-                key={name}
-                autoComplete="new-password"
-                data-amplify-password
-                {...propsCreator('password', 'Password', formOverrides, true)}
-                hasError={!!validationError['confirm_password']}
-                name="password"
-                onBlur={handleBlur}
-              />
+              <React.Fragment key={name}>
+                <PasswordField
+                  autoComplete="new-password"
+                  data-amplify-password
+                  {...propsCreator('password', 'Password', formOverrides, true)}
+                  hasError={!!validationError['confirm_password']}
+                  name="password"
+                  onBlur={handleBlur}
+                />
+                {validationError['password_complexity'] && (
+                  <View data-amplify-sign-up-errors>
```

Comment:
I had to add this `View` in here so the spacing would be correct for the errors.

Reply:
But I didn't want it to show unless there was an error, otherwise it would add this extra blank line after the `password` field.